python/lsst/ts/salobj/topics/remote_command.py

```diff
@@ -277,19 +276,23 @@ async def next_ackcmd(self, ackcmd, timeout=DEFAULT_TIMEOUT, wait_done=True):
         Parameters
         ----------
         ackcmd : `SalInfo.AckCmdType`
-            The command acknowledgement returned by
```

Comment:
Typo above in (new) line 274: "acknowledement" instead of "acknowledgement".  (Also, the first line should end with a period, no?)